### ISSUE TITLE
docker build stuck forever on context creation

### ISSUE THREAD

**Issue by awdAvenger** [macOScontainers/rund#26]
I'm having issues running a simple 'docker build .' command of a very simple Dockerfile:

```
FROM ghcr.io/macoscontainers/macos-jail/ventura:latest

RUN mkdir -p /test
```

Installation has been performed with homebrew as per the readme in the homebrew-repository and I can run containers with 'docker run' like the example in the Readme.

Here is the output when trying 'docker build .':

```
$ docker build  --progress=plain .
#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile:
#1 transferring dockerfile: 212B done
#1 DONE 0.1s
```

And there it's stuck, I tried to wait for 10 mins and if I press CTRL-C after that I get:
```
^CERROR: failed to solve: Canceled: context canceled
```

SIP has been disabled and I'm running MacOS Sonoma:
```
$ sw_vers
ProductName:            macOS
ProductVersion:         14.1.1
BuildVersion:           23B81
```

Here's the docker info while the build is stuck:
```
$ docker info
Client: Docker Engine - Community
 Version:    24.0.7
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /Users/user/.docker/cli-plugins/docker-buildx

Server:
 Containers: 0
  Running: 0
  Paused: 0
  Stopped: 0
 Images: 1
 Server Version: library-import
 Storage Driver: vfs
 Logging Driver: json-file
 Plugins:
  Volume: 
  Network: host null
  Log: json-file local
 Swarm: inactive
 Kernel Version: 23.1.0
 Operating System: arm64 (error determining if containerized)
 OSType: darwin
 Architecture: arm64
 CPUs: 10
 Total Memory: 0B
 Name: Mac-mini.local
 ID: 7082e257-5c11-4ded-bedf-407bb1c5126b
 Docker Root Dir: /private/d/
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

**Comment by awdAvenger** [comment]
I got it to work. 

I was trying this from SSH and didn't notice a popup on the GUI that asked me to enable System Extensions from the recovery OS. After doing that and allowing the macFUSE extension, docker build works as expected.

This should probably be added to the README or similar.

**Comment by slonopotamus** [comment]
I pushed some updates in https://github.com/macOScontainers/homebrew-formula/commit/e559375f12451bf727ca6e2495d50c78e1c108fd. Could you please try updating, restarting containerd + dockerd, and trying again? If things still do not work, you should be able to see logs in `/opt/homebrew/var/log/containerd.log` + `/opt/homebrew/var/log/dockerd.log`.

**Comment by slonopotamus** [comment]
Oh, wait, I have reproduced this. Debugging what is going wrong...

As a temporary workaround, use `DOCKER_BUILDKIT=0 docker build .`

**Comment by slonopotamus** [comment]
Fixed in rund 0.0.6
